<launch>
    <node name="cv_camera" pkg="cv_camera" type="cv_camera_node" />
    <node name="lane_detector" pkg="truggy_vision" type="lane_detector_node.py" />
    <param name="right_turn_detector/cascade_xml" type="str" value="/home/oleg/cascade_classifier/data_v3_working/cascade.xml" />
    <param name="right_turn_detector/pub_topic" type="str" value="/vision/turn_right" />
    <node name="right_turn_detector" pkg="truggy_vision" type="cascade_classifier_node.py" />
</launch>
